app/templates/index.hbs

```diff
@@ -44,7 +46,11 @@
                 <div class="col-xs-12">
                     <h2>{{t "index.subjects.heading"}}</h2>
                     <div class="subjectsList p-lg"> {{!SUBJECTS LIST}}
-                        {{taxonomy-top-list list=model.taxonomies}}
+                        {{#if theme.provider.additionalProviders}}
+                            {{additional-provider-list additionalProviders=theme.provider.additionalProviders}}
```

Comment:
Is there not an usecase for providers having additional providers show up on the search page, while still allowing the submissions (and thus using taxonomy)?
I guess what I'm asking is shouldn't this be using theme.provider.allowSubmissions to show taxonomy? And if so, does showing both look ok (probably not somthing that needs to be looked into until it becomes an actual scenarion)?